pkg/i2gw/notifications/config.go

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package notifications
+
+import (
+	"os"
+
+	"github.com/jedib0t/go-pretty/v6/table"
+	"github.com/jedib0t/go-pretty/v6/text"
```

Comment:
We should be careful importing these non standard libraries, are there more common alternatives ?

Reply:
This is a well used library, so it should not be a concern. One alternative would be https://github.com/olekukonko/tablewriter but i feel the package we are using offers more flexibility and options.

Reply:
ok about this, if it was an APACHE licenses it was all good. 
I checked and it looks like CNCF has a [process](https://github.com/cncf/foundation/blob/main/allowed-third-party-license-policy.md#approved-licenses-for-allowlist) for allowlisting MIT licenses. https://github.com/olekukonko/tablewriter is allowed, you can find it [here](https://github.com/cncf/foundation/blob/main/license-exceptions/CNCF-licensing-exceptions.csv). 

The approval processes would probably take a bit of time, and has some requirements, [here](https://github.com/cncf/foundation/issues/809) is an example of exception request.

Up to you if you want to go through this process, or move to use `github.com/olekukonko/tablewriter` and remove the colors. I'd go with the latter.